build-appdir.sh

```diff
@@ -3,29 +3,20 @@
 set -e
 set -x
 
-# preparations
-mkdir -p appdirs/
 
 #######################################################################
 
 # build AppImageTool AppDir
 APPIMAGETOOL_APPDIR=appdirs/appimagetool.AppDir
 
 rm -rf "$APPIMAGETOOL_APPDIR" || true
-mkdir -p "$APPIMAGETOOL_APPDIR"/usr/{bin,lib/appimagekit}
-cp -f install_prefix/usr/bin/appimagetool "$APPIMAGETOOL_APPDIR"/usr/bin
 
-cp ../resources/AppRun "$APPIMAGETOOL_APPDIR"
-cp install_prefix/usr/bin/appimagetool "$APPIMAGETOOL_APPDIR"/usr/bin/
-cp install_prefix/usr/lib/appimagekit/mksquashfs "$APPIMAGETOOL_APPDIR"/usr/lib/appimagekit/
-cp $(which desktop-file-validate) "$APPIMAGETOOL_APPDIR"/usr/bin/
-cp $(which zsyncmake) "$APPIMAGETOOL_APPDIR"/usr/bin/
+DESTDIR="$APPIMAGETOOL_APPDIR" cmake -DCOMPONENT=appimagetool.AppImage -P cmake_install.cmake
```

Comment:
This command is quite non-intuitive. Please add a comment describing what it does and why we don't just use `make install`.

Reply:
This command does run make install but only for a specific component (appimagetool.AppImage). This component only installs the files required in the AppImage and excludes the rest. 
With `make install` you will get all the targets built in the AppImageKit project installed.

Reply:
I _know_ what it does. But I would like it to be documented for non-CMake-pros.

Reply:
Fixed.